src/main/java/com.springcore.constructor/Test1.java

```diff
@@ -8,7 +8,7 @@
 
 
 public class Test1 {
```

Comment:
Invalid naming conventions. use proper name to indicate the class's functionality